src/labelle/lib/devices/device_manager.py

```diff
@@ -82,8 +81,11 @@ def find_and_select_device(self, patterns: list[str] | None = None) -> UsbDevice
             LOG.debug(dev.device_info)
         dev = devices[0]
         if dev.is_supported:
-            msg = f"Recognized device as \
-                {SUPPORTED_PRODUCTS[SUPPORTED_DEVICE_ID(dev.id_product)]}"
+            foundDeviceConfig: DeviceConfig | None = get_device_config_by_id(
```

Comment:
Do you really need to declare the type of `foundDeviceConfig` (I guess you meant `found_device_config` :-) )?
If the function declares its return type, then I think it should be fine not to declare here.

Reply:
Yeah this was an ugly workaround. Will be obsolete when I fix the other function.
And yes of course I meant snake_case 😉

Reply:
Fixed